### PR TITLE
miden_delivery1

### PR DESCRIPTION
**Pull request by stratisnik**
# Grant Application Checklist

- [ Y] The [`applications/application-template.md`](../applications/application-template.md) or [`applications/research-template.md`](../applications/research-template.md) has been copied, renamed ( "project_name.md") and updated.
- [ Y] An Ethereum address for the payment of the milestones is provided inside the application.  
- [ Y] The total funding amount of the project is up to USD $30k for every grant applications.
- [ Y] The initial PR contains only one commit (squash if needed before submitting your PR).

### CODE REVIEW

File: applications/miden.md [hunk 1]
```diff
@@ -0,0 +1,211 @@
+# VeChain General Grant Application Template
```
Comment (libotony) [inline applications/miden.md:1]: Please do not add application file in milestone delivery.

File: milestone-delivery/deliveries/miden.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+**The Ethereum address has been filled out correctly for this milestone and the delivery is according to the official [milestone delivery guidelines](../).**  
+
+* **PR Link:** https://github.com/vechain/grant-program/pull/135 
+* **Milestone Number:** 5
+
+Please provide a list of all deliverables of the milestone extracted from the initial application and a link to the deliverable itself. Ideally all links inside the below table should include a commit hash, which should be used for testing.
+
+| Number | Deliverable | Link | Notes |
+| ------------- | ------------- | ------------- |------------- |
+| 1. | Smart Contract & Backend |0xa1ada849b79a71529e61615de5c32dfe86b95b7e4547b4dd26bdb011e4c614cd | ...|
```
Comment (libotony) [inline milestone-delivery/deliveries/miden.md:14]: Seems incomplete, please kindly check.

**Review by libotony — CHANGES_REQUESTED** [review]

File: milestone-delivery/deliveries/miden.md (hunk 2 above)
Comment (stratisnik) [reply milestone-delivery/deliveries/miden.md]: @libotony This was updated with all the milestones and deliverables completed. See commit with hash 3319bf89d26e68ecf414930182c78434f98f0f15

Comment (libotony) [reply milestone-delivery/deliveries/miden.md]: @stratisnik Thanks for the update, could you please take a look at my previous comment regarding the application file?

**Comment by stratisnik** [comment]
@libotony are there any updates here?

**Comment by libotony** [comment]
> @libotony are there any updates here?
Hi @stratisnik, I still have a question regarding 1.3. Could you elaborate on it? How can I verify it?

**Comment by stratisnik** [comment]
Hi @libotony ,
As per my previous comments, ownership can be verified via a test transfer of a nominal vthor amount to an address of your choice. Please be more specific with your request if you have any additional comments.

1.3 Signing parameters: As defined under the Master address parameters on the contract. Ownership of the Master Address can be verified via a test transfer.

**Comment by libotony** [comment]
> Hi @libotony , As per my previous comments, ownership can be verified via a test transfer of a nominal vthor amount to an address of your choice. Please be more specific with your request if you have any additional comments.
> 
> 1.3 Signing parameters: As defined under the Master address parameters on the contract. Ownership of the Master Address can be verified via a test transfer.

Sorry, I didn't find your previous comment, could you point it out? My question is regarding `Signing parameters as defined within the contract` as a delivery,  what parameters do you define and what the purpose of it and how do I verify the parameters, basically I need more info regarding the content of the delivery.

**Comment by stratisnik** [comment]
> https://explore-testnet.vechain.org/accounts/0xd9eed7e3d56cb6e36316413a02537172a2184f4b/

Hi @libotony thanks for your prompt response. The signing parameter mentioned there is basically the Master address specified on the contract which can be viewed on the vechain explorer. Basically what it means is that the eligibility to issue NFTs within the contract falls under the master agreement which is owed by miden. The ownership of the Master address can be verified through the method specified above.

**Review by libotony — APPROVED** [review]

**Review by NecoSherry — APPROVED** [review]

**Comment by NecoSherry** [comment]
Thank you for delivering milestone 1. We will promptly initiate the payment process for milestones 2. If there are any updates, we will notify you by email. Thank you in advance for your patience and cooperation.

**Comment by stratisnik** [comment]
Dear all,

We just noticed that the ETH wallet we submitted at the beginning of the
grant has been compromised. Could you please let me know what is the
process of receiving funds in another wallet?

Kind regards,
NS

**Pull request merged**